lib/api/element-commands.js

```diff
@@ -340,8 +340,13 @@ module.exports = function(client) {
   function CommandAction(using, value, protocolAction, args, callback, originalStackTrace) {
     events.EventEmitter.call(this);
 
-    var $this = this;
-    var el = Protocol.element(using, value, function(result) {
+    var self = this;
+    var el = Protocol.elements(using, value, function(result) {
+
+      if (result.status === 0 && result.value.length === 0) {
```

Comment:
How about if we do `if (result.status === 0 && result.value && result.value.length > 0) {` in the `if` below (instead of the `else` branch?